.vscode/settings.json

```diff
@@ -2,13 +2,11 @@
 {
   "clang-format.style": "file",
   "clang-format.formatOnSave": true,
```

Comment:
That definitely sounds fair, it looks like the `"search.exclude"` rule exists for this.